client/hotrod-client-new/src/main/java/org/infinispan/client/hotrod/impl/operations/StatsOperationsFactory.java

```diff
@@ -72,6 +73,25 @@ public <K, V> HotRodOperation<Map<K, V>> newGetAllBytesOperation(Set<byte[]> key
       return new StatisticOperation<>(super.newGetAllBytesOperation(keys), statistics);
    }
 
+   @Override
+   public HotRodOperation<Boolean> newPutStreamNextOperation(int id, boolean lastChunk, ByteBuf valueBytes, Channel channel) {
+      var op = super.newPutStreamNextOperation(id, lastChunk, valueBytes, channel);
+      if (lastChunk) {
```

Comment:
Is it OK to discard the times to transfer + append the data? TBH I think it is OK for us to be inaccurate in the timing for this case.

Reply:
True, however capturing the timing here is not simple as it can span multiple operations. TBH, to properly capture that is not simple as we would have to have some sort of state in the StatsOperationFactory and having expiration for those values in case the put stream was never completed. Like you said, I don't know if it is worth it or not.

Reply:
I will try to think if there is a better way or not.